github/pulls_test.go

```diff
@@ -67,6 +68,60 @@ func TestPullRequestsService_Get(t *testing.T) {
 	}
 }
 
+func TestPullRequestService_GetRawPatch(t *testing.T) {
+	setup()
+	defer teardown()
+	const rawStr = "@@patch content"
+
+	mux.HandleFunc("/repos/o/r/pulls/1", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", mediaTypeV3Patch)
+		fmt.Fprint(w, rawStr)
+	})
+
+	ret, _, err := client.PullRequests.GetRaw("o", "r", 1, RawOptions{Patch})
```

Comment:
Client code will not be able to write `github.RawOptions{github.Patch}`, because that'd cause a vet issue of composite literal using unkeyed fields.

They'd have to write `github.RawOptions{Type: github.Patch}`.

On the one hand, keeping the `RawOptions` struct means we can add additional options in the future, but on the other hand, it's more verbose than just passing a `RawType` parameter without a struct.

Just making an observation here.